src/main/java/io/supertokens/webserver/api/totp/VerifyTotpDeviceAPI.java

```diff
@@ -70,7 +71,7 @@ protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws I
             // Also return a retryAfter value:
             resp.addHeader("Retry-After", Integer.toString(e.retryInSeconds));
             super.sendJsonResponse(429, result, resp); // 429 (Too Many Requests)
```

Comment:
> how many tries were done, and what the limit

Since rate limiting is active, tries done == max limit will always be the case. I don't think we need to return any of those.